pkg/overlay/statdb_test.go

```diff
@@ -25,6 +25,8 @@ func getRatio(success, total int64) (ratio float64) {
 }
 
 func TestStatDB(t *testing.T) {
+	// TODO(team green): remove the skip when implementing V3-1843
+	t.Skip("require disqualification implementation")
```

Comment:
next time add the issue number in the same `skip` comment.
No need to change it in this PR, but I think that's better to easily find a reference if somebody read the code, without having to dig in the git history.

Reply:
okay, I'll just go ahead and add the PR number